liboptv/tests/CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ if (UNIX)
     set(LIBS ${LIBS} -lm -pthread)
 endif (UNIX)
 
+if (APPLE)
```

Comment:
The problem is not liboptv, but rather check - this library is somehow not
found by tests and it seems to relate to the way cmake interacts with it. I
guess it is somewhere in my incompetence with cmake build process. I do not
need everyone to have it, of course, we can just ad it somewhere to docs
for other poor Mac users.





On Thursday, September 3, 2015, Yosef Meller <notifications@github.com>
wrote:

> In liboptv/tests/CMakeLists.txt
> <https://github.com/yosefm/openptv/pull/16#discussion_r38615756>:
>
> > @@ -35,6 +35,11 @@ if (UNIX)
> >      set(LIBS ${LIBS} -lm -pthread)
> >  endif (UNIX)
> >
> > +if (APPLE)
>
> I'm pretty sure this shouldn't be necessary. Perhaps you have two versions
> of liboptv installed, one in /usr/ and one in /usr/local/ ?
>
> If so, I suggest to clean up the /usr/ version and then this hard-coded
> path will not be necessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yosefm/openptv/pull/16/files#r38615756>.
>